addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/BrowserBasedAuthenticationMethodType.java

```diff
@@ -512,6 +534,21 @@ public AuthenticationMethod loadMethodFromSession(Session session, int contextId
                 // Ignore
             }
         }
+
+        try {
+            List<AuthenticationStep> loaded =
+                    session
+                            .getContextDataStrings(
+                                    contextId, RecordContext.TYPE_AUTH_METHOD_FIELD_4)
+                            .stream()
+                            .map(BrowserBasedAuthenticationMethodType::decodeStep)
+                            .filter(Objects::nonNull)
+                            .toList();
+            method.setAuthenticationSteps(loaded);
+        } catch (Exception e) {
+            LOGGER.error("An error occurred while loading the data:", e);
```

Comment:
Space after colon?

Reply:
Or does it endup being the next line anyway because it's the whole exception?

Reply:
The latter.